squidDrone/src/i2c/i2c_interface.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef SRC_I2C_INTERFACE_HPP_
+#define SRC_I2C_INTERFACE_HPP_
+
+#include "i2c_status.hpp"
+#include <cstdint>
+#include <vector>
+#include <tuple>
+
+namespace i2c {
+
+  /**
+   * @brief Interface of I2C package
+   * 
+   */
+  class I2CInterface {
```

Comment:
I couldn't find the model in the wiki, please point me to a link?

Reply:
It was not added so far. I added it now: https://github.com/elheck/Squiddrone/wiki/I2C-Class